index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>ERNEST Portfolio Website</title>
+</head>
+<body>
+    <!-- defining the header and wraping it in divs and will contain the logo-->
+    <div class="main_container">
+        <div class="inside_container">
+            <header class="main_header">
+              <nav class="navbar">
+                <a href="#" class="my-portfolio-logo">
+                    <img src="./assets/MyLogo.png" alt="My-logo"/>
+                </a>
+                <a><img src="./assets/menu.png" alt="menu-icon"/></a>
+              </nav>
+            </header>
+    <!-- Defining the main section for the description of the developer -->
+            <main class="main_section">
```

Comment:
- [ ] Kindly ensure to use the `<section>` tag for this section as indicated in the previous review. This will ensure the proper use of HTML semantic tags